src/main/java/space/obminyashka/items_exchange/controller/request/AdvertisementFindThumbnailsRequest.java

```diff
@@ -0,0 +1,30 @@
+package space.obminyashka.items_exchange.controller.request;
+
+import io.swagger.v3.oas.annotations.Parameter;
+import lombok.*;
+
+import javax.validation.constraints.PositiveOrZero;
+import java.util.UUID;
+
+import static space.obminyashka.items_exchange.util.ResponseMessagesHandler.ValidationMessage.*;
+
+@Getter
+@Setter
+public class AdvertisementFindThumbnailsRequest {
+
+    @Parameter(name = "page", description = "Results page you want to retrieve (0..N). Default value: 0")
+    @PositiveOrZero(message = "{" + INVALID_NOT_POSITIVE_ID + "}")
+    private int page = 0;
+
+
+    @Parameter(name = "size", description = "Number of records per page. Default value: 12")
+    @PositiveOrZero(message = "{" + INVALID_NOT_POSITIVE_ID + "}")
+    private int size = 12;
+
+    @Parameter(name = "subcategoryId", description = "ID of existed subcategory for searching same advertisements")
+    private Long subcategoryId;
```

Comment:
I would ask you to put Nullable annotation above that parameter and excludeAdvertisementId. They would allow us not to miss that case